### DISCUSSION TITLE
How can I change the header of the monthly calendar to the French version?

### DISCUSSION THREAD

**Discussion by Artlyve** [vkurko/calendar#271] [Q&A] [answered]
Hello ,
In timeGridWeek the header is correctly but in dayGridMonth the list starte with the wrong days.
There's a mismatch on the header I tried to do this but it had no effect:
		_firstDay:'1',
		locale:'fr',_
		
timeGridWeek view:
![week](https://github.com/vkurko/calendar/assets/47179772/0c89c6c6-3698-4db9-8e13-903f749a512b)
dayGridMonth view :
![image](https://github.com/vkurko/calendar/assets/47179772/4ee7eec3-8db1-4659-a1bc-1c6a1923d3ca)

is there a solution for this ?

**Comment by vkurko** [comment, marked as answer]
Hello,

The [firstDay](https://github.com/vkurko/calendar?tab=readme-ov-file#firstday) option accepts number, not string. So try `firstDay: 1`.

**Reply by Artlyve** [reply]
thank you for your answer !
